grin/tests/framework/mod.rs

```diff
@@ -189,6 +189,22 @@ impl LocalServerContainer {
 			seeds = vec![self.config.seed_addr.to_string()];
 		}
 
+		let mut plugin_config = pow::types::CuckooMinerPluginConfig::default();
+		let mut plugin_config_vec: Vec<pow::types::CuckooMinerPluginConfig> = Vec::new();
+		plugin_config.type_filter = String::from("mean_cpu");
```

Comment:
this should rarely be executed in practice (mostly for testing), but shouldn't this be 'mean_compat_cpu' to avoid segfaults on older devices?